### DISCUSSION TITLE
Allow models to retrieve URLs from the Internet

### DISCUSSION THREAD

**Discussion by hassankhan** [janhq/jan#2188] [Feature Ideas]
I'd really love to be able to ask for a summary of a URL, however I couldn't seem to find anything in the documentation and the app itself gives me this when I ask it:

<img width="552" alt="image" src="https://github.com/janhq/jan/assets/1781985/5283342b-b8bb-47f7-8dcb-ace59f0a2a88">

Would the best way to achieve this be a Jan extension?

**Comment by lv333ming** [comment]
This is a very practical idea, and I hope that large models can continue to improve knowledge base capabilities.
